captura/views.py

```diff
@@ -224,7 +227,153 @@ def create_estudio(request):
+    values, and redirect the user to it in order to fill it.
+
+    Returns
+    ----------
+    GET:
+        On succes returns HTTP 200 with captura/integrante_form.html
+        template rendeered.
+
+        On error returns HTTP 404
+
+    POST:
+        On succes returns HTTP 301 redirect to the integrante table
+        or to itself in case it associates a new model to an user.
+
+        On error returns HTTP 404
+
+    """
+    integrante_form = None
+    if request.method == 'POST':
+        integrante = get_object_or_404(Integrante, pk=id_integrante)
+        integrante_form = IntegranteForm(request.POST, instance=integrante)
+        if integrante_form.is_valid():
+            integrante_form.save()
+            rol = integrante_form.cleaned_data['Rol']
+            if rol == integrante_form.OPCION_ROL_ALUMNO:
+                alumnos = Alumno.objects.filter(integrante=integrante)
+                if alumnos:
+                    alumno = alumnos[0]
+                    alumno_form = AlumnoForm(request.POST, instance=alumno)
+                    if alumno_form.is_valid():
+                        alumno_form.save()
+                else:
+                    escuela = Escuela.objects.all().first()
+                    Alumno.objects.create(integrante=integrante, escuela=escuela)
+                    return redirect(reverse('captura:integrante',
+                                            kwargs={'id_integrante': id_integrante}))
+
+            elif rol == integrante_form.OPCION_ROL_TUTOR:
+                tutores = Tutor.objects.filter(integrante=integrante)
+                if tutores:
+                    tutor = tutores[0]
+                    tutor_form = TutorForm(request.POST, instance=tutor)
+                    if tutor_form.is_valid():
+                        tutor_form.save()
+                else:
+                    Tutor.objects.create(relacion=integrante_form.cleaned_data['Rol'],
+                                         integrante=integrante)
+                    return redirect(reverse('captura:integrante',
+                                            kwargs={'id_integrante': id_integrante}))
+
+            return redirect(reverse('captura:integrantes',
+                                    kwargs={'id_familia': integrante_form.instance.familia.pk}))
+    forms = {}
+    integrante = Integrante.objects.get(pk=id_integrante)
+    rol_integrante = 'ninguno'
+    rol_disabled = False
+
+    alumnos = Alumno.objects.filter(integrante=integrante)
+    if alumnos:
+        alumno = alumnos[0]
+        forms['form_alumno'] = AlumnoForm(instance=alumno)
+        rol_integrante = 'alumno'
+        rol_disabled = True
+
+    tutores = Tutor.objects.filter(integrante=integrante)
+    if tutores:
+        tutor = tutores[0]
+        forms['form_tutor'] = TutorForm(instance=tutor)
+        rol_integrante = 'tutor'
+        rol_disabled = True
+
+    if integrante_form:
```

Comment:
Using it as none is used at the end of the function, this is required in order to be able to pass the form errors to the user.